### PR TITLE
[X86][MC] Support case insenstive for `dfv,sf,cf,of,zf` for CCMP/CTEST

### PR DESCRIPTION
**Pull request by FreddyLeaf**

### CODE REVIEW

File: llvm/test/MC/X86/apx/ccmp-att.s [hunk 1]
```diff
@@ -1211,3 +1211,9 @@
 # CHECK: ccmpeq {dfv=of} %r9, %r15
 # CHECK: encoding: [0x62,0x54,0xc4,0x04,0x39,0xcf]
          ccmpeq {dfv=of} %r9, %r15
+
+## Case Sensitive
```
Comment (KanRobert) [inline llvm/test/MC/X86/apx/ccmp-att.s:1215]: Sensitive -> sensitive

File: llvm/test/MC/X86/apx/ccmp-intel.s [hunk 2]
```diff
@@ -1208,3 +1208,9 @@
 # CHECK: ccmpe {dfv=of} r15, r9
 # CHECK: encoding: [0x62,0x54,0xc4,0x04,0x39,0xcf]
          ccmpe {dfv=of} r15, r9
+
+## Case Sensitive
```
Comment (KanRobert) [inline llvm/test/MC/X86/apx/ccmp-intel.s:1212]: ditto

File: llvm/test/MC/X86/apx/ccmp-att.s (hunk 1 above)
Comment (FreddyLeaf) [reply llvm/test/MC/X86/apx/ccmp-att.s]: a46ebb7

**Review by KanRobert — APPROVED** [review]
LGTM

**Pull request merged**